Command/CatalogMedia.php

```diff
@@ -65,29 +83,34 @@ public function __construct(ResourceConnection $resource, File $file)
      */
     protected function configure()
     {
-        $this->setName('sivaschenko:catalog:media')
-            ->setDescription('Get information about catalog product media')
-            ->addOption(
-                self::INPUT_KEY_REMOVE_UNUSED,
-                'r',
-                InputOption::VALUE_NONE,
-                'Remove unused product images'
-            )->addOption(
-                self::INPUT_KEY_REMOVE_ORPHANED_ROWS,
-                'o',
-                InputOption::VALUE_NONE,
-                'Remove orphaned media gallery rows'
-            )->addOption(
-                self::INPUT_KEY_LIST_MISSING,
-                'm',
-                InputOption::VALUE_NONE,
-                'List missing media files'
-            )->addOption(
-                self::INPUT_KEY_LIST_UNUSED,
-                'u',
-                InputOption::VALUE_NONE,
-                'List unused media files'
-            );
+        $this->setName('sivaschenko:catalog:media')->setDescription(
```

Comment:
When you have to wrap multiple function calls to multiple line, put each call on its own line, including the first one:
```suggestion
        $this
            ->setName('sivaschenko:catalog:media')
            ->setDescription('Get information about catalog product media')
            ->addOption(
                self::INPUT_KEY_REMOVE_UNUSED,
                'r',
                InputOption::VALUE_NONE,
                'Remove unused product images'
            )->addOption(
            ...
```